src/components/RedeemUI/index.tsx

```diff
@@ -0,0 +1,155 @@
+import clsx from 'clsx';
```

Comment:
I extracted this part as a component and re-used in two places.

Reply:
@crypto-engineer - to check, this isn't new code is it? Just moved?

Reply:
Yes